docs/en/Running-Alluxio-On-Docker.md

```diff
@@ -57,12 +81,19 @@ $ docker run -d --net=host alluxio master
 
 We need to tell the worker where to find the master. Set the `ALLUXIO_MASTER_HOSTNAME`
 environment variable to your machine's hostname when launching the worker Docker container.
+To enable short-circut reads, share the ramdisk with `-v /mnt/ramdisk:/mnt/ramdisk`, and
```

Comment:
circut -> circuit

Also, maybe describe what the `-v` command is doing? it is sharing the host ramdisk into a path inside the container.